hw2/03-count.js

```diff
@@ -1,3 +1,18 @@
 // Add your code here
 
-input.addEventListener('keydown', handleKeyDown);
+// reference-----https://tomekdev.com/posts/highlight-text-in-javascript
+
+let paragraphID = document.getElementById("paragraph");
+let inputID = document.getElementById("inputLabel");
+
+inputID.addEventListener("input", (event) => {
+  const searchText = event.target.value;
+  const regex = new RegExp(searchText, "gi");
```

Comment:
The requirements aren't the _most_ clear on this point, but on closer reading they seem to suggest our pattern matching should target entire words and not partial matches. I'm commenting super late, so I definitely don't think you're obligated to change this, but in case you're curious, both Ian and I used the `\b` metacharacter to target whole-word matches only.

Reply:
I have applied the changes, thanks for the feedback.